src/analysis/complex/basic.lean

```diff
@@ -258,6 +258,26 @@ lemma isometry_of_real : isometry (coe : ℝ → ℂ) := of_real_li.isometry
 /-- Continuous linear map version of the canonical embedding of `ℝ` in `ℂ`. -/
 def of_real_clm : ℝ →L[ℝ] ℂ := of_real_li.to_continuous_linear_map
 
+/-- The only continuous ring homomorphisms from `ℂ` to `ℂ` are the identity and the complex
+conjugation, -/
```

Comment:
```suggestion
conjugation. -/
```

Reply:
Thanks a lot for all the comments and the improvements. I thought my proofs were pretty short but yours are so much more efficient 👍

Reply:
Haha, your original proofs were already quite efficient and I just know more tricks to compress them :)